go.mod

```diff
@@ -168,19 +170,47 @@ replace golang.org/x/net v0.0.0-20211112202133-69e39bad7dc2 => golang.org/x/net
 
 replace golang.org/x/net v0.0.0-20220722155237-a158d28d115b => golang.org/x/net v0.7.0
 
-replace golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c => golang.org/x/text v0.3.7
+replace golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c => golang.org/x/text v0.3.8
```

Comment:
Just curious if there is something preventing us from replacing these with golang.org/x/text v0.9.0 which is the version we specify in our dependencies now.

Reply:
I did notice v0.9.0 to be the most recent one but I went with the minimum requirement listed in the ticket. I think we should keep things in harmony so I'll change these to v0.9.0 as well!